pubpub/pubpub
# enhancement: one-paragraph abstracts only

implements #1876

## _Test Plan_
+ open a spub
+ navigate to the 'submission' tab of that spub
+ give the spub an abstract
+ try to make it multi-paragraph (i.e. hit 'enter') and you should be unsuccessful
+ try to make it multi-line (i.e. hit 'shift + enter') and you should have no problem
+ try to paste in an image or an `H2` copied from elsewhere, and you should be prevented

## Todo
+ is this the behavior and limitation we want? Or do we want to limit the doc to `paragraph+` and then wrap them in a single paragraph on save?
